### DISCUSSION TITLE
Not enough privileges to submit your translation on weblate

### DISCUSSION THREAD

**Discussion by VladimirLevadnij** [uxbox/uxbox#1596] [General]
Hello. I want to help with the translation of interfaces into Russian on the site https://hosted.weblate.org/projects/penpot/

I'm trying to suggest a translation, but I'm getting a message that I don't have the required privileges.

Can you please tell me how can I get access so that I can participate in the interface translation work?

![image](https://user-images.githubusercontent.com/20191527/154168703-e4049882-55bc-4e7b-9e24-4bb73ff89f0f.png)

**Comment by soultipsy** [comment]
Hello @VladimirLevadnij 
You can find help here:https://help.penpot.app/contributing-guide/translations/

To start translating at Penpot:
1.Open a github issue giving details about the language you want to translate (language), the type of translation (new language, new translation or change an existing translation) and your Weblate user.
2.If everything is correct we will get back to you providing you permissions to the actions needed.

**Reply by VladimirLevadnij** [reply]
Thanks!
